token.sol

```diff
@@ -237,13 +235,14 @@ contract token is safeMath, module, announcementTypes {
         if ( _from != msg.sender ) {
             allowance_[_from][msg.sender].amount = safeSub(allowance_[_from][msg.sender].amount, _amount);
             allowance_[_from][msg.sender].transactionCount++;
-            EAllowanceUsed(msg.sender, _from, _amount);
+            AllowanceUsed(msg.sender, _from, _amount);
         }
         if ( isContract(_to) ) {
             bytes memory data;
             transferToContract(_from, _to, _amount, data);
         } else {
-            require( transfer_( _from, _to, _amount, true) );
+            transfer_( _from, _to, _amount, true);
+            Transfer(_from, _to, _amount);
```

Comment:
I fire event inside `transferToContract` function.
https://github.com/CORIONplatform/solidity/blob/transfer-fix/token.sol#L312
Its two different events for contract transfer and normal address transfer.
Ok?